pandas/core/common.py

```diff
@@ -463,12 +463,14 @@ def convert_to_list_like(
     values: Union[Scalar, Iterable, AnyArrayLike]
 ) -> Union[List, AnyArrayLike]:
     """
-    Convert list-like or scalar input to list-like. List, numpy and pandas array-like
-    inputs are returned unmodified whereas others are converted to list.
+    Convert list-like or scalar input to list-like.
+
+    List, numpy and pandas array-like inputs are returned unmodified
+    whereas others are converted to list.
     """
     if isinstance(values, (list, np.ndarray, ABCIndex, ABCSeries, ABCExtensionArray)):
         # np.ndarray resolving as Any gives a false positive
-        return values  # type: ignore[return-value]
+        return cast(Union[List, AnyArrayLike], values)
```

Comment:
The comment above indicates that this ignore can be removed after we have numpy types. no need to add cast yet. While numpy types resolve to `Any` and np.ndarray is part of AnyArrayLike we may have false positives.

Reply:
Ok, thank you for the clarification.
So, probably this PR should be closed?

Reply:
the changes to the docstring are ok, but I'll close this for now.